lib/src/rules/directives_ordering.dart

```diff
@@ -249,6 +347,11 @@ class _Visitor extends SimpleAstVisitor {
         .forEach(reportDirective);
   }
 
+  Iterable<ExportDirective> _getExportDirectives(CompilationUnit node) =>
+      node.directives
+          .where(_isExportDirective)
+          .map((e) => e as ExportDirective);
```

Comment:
I think so, If I delete it and run $pub run test, A lot of tests start to fail and I got this error message:

```
Caused by type 'WhereIterable<Directive>' is not a subtype of type 'Iterable<ExportDirective>' of 'function result' where
  WhereIterable is from dart:_internal
  Directive is from package:analyzer/dart/ast/ast.dart
  Iterable is from dart:core
  ExportDirective is from package:analyzer/dart/ast/ast.dart
```